components/card-section-ab1.html

```diff
@@ -1,5 +1,14 @@
-
-
+<style>
+    .secondary {
```

Comment:
Add this style in a scss file to avoid the repetitions in the other similar components.

Reply:
Copy that.